tools/hangar/__snapshots__/test_corpus/valid/bring_jsii.w.ts.snap

```diff
@@ -177,14 +177,18 @@ class $App extends $AppBase {
     }
   }
 }
-
-new $App().synth();"
+new $App().synth();
+"
 `;
 
-exports[`wing compile -t tf-aws > proc.5d209e0c3acca40c825662cab73c204803d9dd9f7903de90d0da6dc99ea7fa35/index.js 1`] = `
-"async handle(m) { const { greeting } = this; {
-  {((cond) => {if (!cond) throw new Error(\`assertion failed: '(greeting === \\"Hello, wingnuts\\")'\`)})((greeting === \\"Hello, wingnuts\\"))};
-} };"
+exports[`wing compile -t tf-aws > proc.f9fd45e979c44914dac14967e8ad3a3b4c5beb85154a204a119e273326d0f3c0/index.js 1`] = `
+"async handle(m) {
+  const { greeting } = this;
+  {
+    {((cond) => {if (!cond) throw new Error(\`assertion failed: '(greeting === \\"Hello, wingnuts\\")'\`)})((greeting === \\"Hello, wingnuts\\"))};
+  }
+};
```

Comment:
@Chriscbr do we really need this `;`? Feels like it's not supposed to be there.

Reply:
I don't think the last semicolon in this snapshot is needed, we omit similar ones when writing `async handle(m)` snippets in the SDK tests.

It sounds plausible that the semicolon at the end of the `assert` macro line could also be omitted, feel free to change as you see fit